src/latexdocvisitor.cpp

```diff
@@ -52,6 +52,32 @@ static const char *getSectionName(int level)
   return secLabels[std::min(maxLevels-1,l)];
 }
 
+static void insertDimension(TextStream &t, QCString dimension, const char *percentString)
+{
+  // dimensions for latex images can be procentual, in this case they need some extra
+  // handling as the % symbol is used for comments
+  int i = dimension.find('%');
+  if (i > 0)
+  {
+    QCString value = dimension.left(i).stripWhiteSpace();
+    int number = 0;
+    if (sscanf(value.data(),"%d",&number) != 0)
+    {
+      t << (double)number / 100;
+      t << "\\text";
+      t << percentString;
```

Comment:
Maybe good to mention in the documentation (with the `\image` command) that in case of a percentage the percentage is taken relative to the current textwidth / textheight